src/grid3d_maps/forward_models/grid3d_aggregate_map.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from ert import (
+    ForwardModelStepDocumentation,
+    ForwardModelStepJSON,
+    ForwardModelStepPlugin,
+)
+
+from grid3d_maps.aggregate.grid3d_aggregate_map import DESCRIPTION
+
+
+class Grid3dAggregateMap(ForwardModelStepPlugin):
+    def __init__(self):
```

Comment:
```suggestion
    def __init__(self) -> None:
```
I think all of these initializers are missing this type annotation